pkg/install/helm/cli.go

```diff
@@ -55,6 +55,47 @@ func NewCLI(binary string, kubeconfig string, kubeContext string, timeout time.D
 	}, nil
 }
 
+func (c *cli) BuildChartDependencies(chartDirectory string, flags []string) (err error) {
+	command := []string{
+		"dependency",
+		"build",
+	}
```

Comment:
Can you move this declaration down to where it's needed? I lost track of what "command" was (granted, I read code at like 9000% zoom level ^^)

Reply:
Good note, done.